cmd/databases.go

```diff
@@ -142,6 +142,7 @@ var (
 
 	databaseListUsers = cli.Command{
 		Name:     "database-list-users",
+		Aliases:  []string{"database-users-list"},
```

Comment:
**nitpick (non-blocking)**: as "database-users-" if our wanted prefix, shouldn't it be the name of the command? and the old name the alias

Reply:
Yep you're right